README.md

```diff
@@ -60,10 +77,43 @@ A lesson is particularly well-suited to The Carpentries Lab if:
 * To request a review, [open an issue in this repository][new-issue] and respond to the prompts in the _Review submission_  issue template
 * The editor will [assess the submission for fit and overall quality][editor-guide], and assign reviewers.
 * The reviewers will use [The Carpentries checklists and guidelines][reviewer-guide] to provide feedback on the lesson.
-* Once accepted, the lesson will be transfered to [The Carpentries Lab GitHub organization][lab-gh-org], updated to use The Carpentries Lab theme, and be listed on The Carpentries Lab website.
+* Once accepted, the lesson will be transfered to [The Carpentries Lab GitHub organization][lab-gh-org],
+  updated to use The Carpentries Lab theme,
+  given a URL under `https://carpentries-lab.org/`,
+  and listed on [The Carpentries Lab website][lab-home].
+* If you choose to also submit the lesson for publication in [JOSE][jose],
+  the accompanying paper in your lesson repository will be reviewed separately
+  by their team (see below).
 
 _This process is inspired by the software review from [rOpenSci][ropensci]_
 
+
+## Joint Review with JOSE
+
+When submitting their lesson for review in The Carpentries Lab,
+authors will have the option to also submit to the review process for
```

Comment:
should we specify here that the review with JOSE happens after the review with Carpentries Lab?
I'm assuming that, ideally, the JOSE reviewers and editors would want to see the types of comments/feedback and improvements made to the lesson following the carpentries lab review.

Reply:
Great point- I also wonder what would happen if a lesson was rejected from inclusion in the Carpentries Lab. Would that mean they can still submit to JOSE? Or are we saying that if you choose to submit via the Incubator -> Lab pipeline, acceptance to the Lab is a prerequisite for JOSE submission?